source/plugins/go/src/oms.go

```diff
@@ -1171,6 +1254,8 @@ func PostDataHelper(tailPluginRecords []map[interface{}]interface{}) int {
 		}
 
 		logEntry := ToString(record["log"])
+		//filter loglevel and define here
+		//logLevel := ToString(record["LogLevel"])
```

Comment:
are we plan to enable logLevel code when this PR ready to merge?

Reply:
yes Amol will be doing that after private preview